index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'objection-unique' {
+  import { Model } from 'objection';
+
+  export default function (options: {
+    identifiers: Array<string | string[]>;
```

Comment:
Oh, this is odd. `identifiers` is not technically required, since it defaults to `["id"]`:

```javascript
  options = Object.assign({
    identifiers: ['id']
  }, options);
```

Otherwise looks good, works fine.

Reply:
Yeah, you are right. If you don't set the `options.identifiers`, the validation I talked before won't fail because it has the default value `['id']`. It just will fail if you set it as `[]` or not set the `fields` option. So you can change `identifiers` to be optional, but keep the `fields` as required, mb :)

Reply:
Hello, I made the identifiers optional